### PR TITLE
Ergonomics

### PR DESCRIPTION
**Pull request by HeroicKatora**
Closes: #233

### CODE REVIEW

File: fuzz/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+# Fuzzing i686 on native `x86_64` linux
```
Comment (nabijaczleweli) [inline fuzz/README.md:2]: The codification of i686 vs `x86_64` is inconsistent here (also linux might be better off as Linux)

Comment (HeroicKatora) [reply fuzz/README.md:2]: It's consistent with the names in `rustup target list`. I'll change the capital of linux, good point.

Comment (nabijaczleweli) [reply fuzz/README.md:2]: I mostly meant the code blocks around `x86_64` and lack thereof around i686

Comment (HeroicKatora) [reply fuzz/README.md:2]: Ohhhh. That makes sense then xd

Comment (HeroicKatora) [reply fuzz/README.md:2]: Just checked that Github's markdown flavor can render inline code in titles so I'll add it to both. I vaguely remember that jekyll (the Github pages app) could not.

**Comment by HeroicKatora** [comment]
@nabijaczleweli Thanks for the review.

**Pull request merged**